src/spoke/BalanceSheet.sol

```diff
@@ -191,6 +194,9 @@ contract BalanceSheet is Auth, Multicall, Recoverable, IBalanceSheet, IBalanceSh
         } else if (shareQueue.delta > shares) {
             shareQueue.delta -= shares;
             shareQueue.isPositive = true;
+            if (shareQueue.delta == 0) {
+                shareQueue.isPositive = false;
+            }
```

Comment:
This branch seems to be unreachable because the upper check is `shareQueue.delta > uint128(shares)` such that the delta can never be zero. I suppose you intended to move this `if` branch to the upper level `if else`?

Reply:
Ah correct! Then that can be removed